test/unit/util.test.ts

```diff
@@ -65,3 +71,32 @@ test('Get full URL', async(t) => {
             'https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1',
             'Full Url when base and url both strtas with http/s');
 })
+
+test('Custom flavour path', async(t) => {
+
+    // checks in current working directory.
+    const tmpObj = tmp.fileSync({ postfix: '.js' });
+    process.chdir('/tmp');
```

Comment:
No, you should change directory based on values of `tmpObj`. You have no garanty that this is in `/tmp`.